lib/grinder_sdk.dart

```diff
@@ -44,7 +44,7 @@ class Dart {
   /// Returns the stdout.
   static String run(String script,
       {List<String> arguments : const [], bool quiet: false,
-       String packageRoot, String workingDirectory, int vmNewGenHeapMB,
+       String packageRoot, RunOptions runOptions, int vmNewGenHeapMB,
```

Comment:
I had the same thoughts, tried `options` first but had the impression this could be confusing.